src/third/react-native/KeyboardAvoidingView.js

```diff
@@ -0,0 +1,251 @@
+/* @flow strict-local */
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
This should be updated; for one thing, with this file in a different source tree, there is a LICENSE file in the root directory but it's a different license.

Relatedly, we'll want to add an entry in `docs/THIRDPARTY`.

I can make those edits before merging.

Reply:
In the commit message, let's mention the exact RN version, and path in its tree, that this file is copied from. That makes it easy to compare, and also to look for upstream changes in the future.

Reply:
> I can make those edits before merging.

Cool, thanks!